### ISSUE TITLE
ct.js "Package project to zip" makes incomplete archives

### ISSUE THREAD

**Issue by CosmoMyzrailGorynych** [ct-js/ct-js-old#116]
**Describe the bug**
See "to reproduce".
The bug seems to be exclusive to Windows. Maybe it has something to deal with `archiver`, maybe with `fs-extra`, or maybe with streaming on Windows in general.

**To Reproduce**
1. Open a project with numerous textures, e.g. [SpaceShooter_tutorial.ict](https://github.com/ct-js/ct-js/blob/develop/src/examples/SpaceShooter_tutorial.ict).
2. Click the hamburger menu at top-left > Package project to .zip
3. Get an archive with a random content
4. Repeat and see that archives are generated with different sets of files.

**Expected behavior**
Ct.js should pack all the project's files.

**Versions:**
 - Windows 10. Could not reproduce it on Fedora 30.
 - ct.js version 1.0.1

**Comment by CosmoMyzrailGorynych** [comment]
This one is interesting. For some reason, the temporary directory `zippedProject` remained empty when it should be filled with copied files. Probably a malfunction in `fs.emptyDir`. `fs.remove` didn't help either. It happened exactly each second attempt to zip the project.

The solution was to create individual, unique temporary folders for each zip procedure.

TL;DR: hello windows
